### ISSUE TITLE
only 10 posts collectable 

### ISSUE THREAD

**Issue by KeyGirl93** [strohne/Facepager#47]
![facepager](https://cloud.githubusercontent.com/assets/23215647/19932217/8b4731a4-a10f-11e6-9384-9a1dd0865979.PNG)

I want to analyse the feed of a specific facebook group. But I can only download the last 10 posts. Furthermore if I set the parameters 'since' and 'until', facepager is answering with empty. Where is the mistake?

One extra question: I want to analyze posts with all related likes, shares, comments and answers with all other reactions to this comments for a specific period inlcuding links, photos and videos. Is it possible to collect all this data with fetching one time?

**Comment by dorvak** [comment]
Please have a look at the Facebook API Documentation, as it will answer all of your questions (pagination, facetted queries etc.)

**Comment by KeyGirl93** [comment]
I have done this but there was no solution for my problem. Now a few weeks later it is possible to catch more posts without changing the strategy. anyway it works, thx.

Now I have two more problems:
1. Facepager is randomly skipping some posts and more comments while fatching data. I realized that while comparing the data I got through facepager with the browser-facebook-version.
2. Is there a special key with which I can catch the answers to comments under a facebook-posts? At the moment Facepager is just collecting posts and comments but not answers to comments (or i didn't found them).
